src/components/Card/_Card.scss

```diff
@@ -130,3 +128,26 @@ header p:nth-child(1) {
   text-decoration: none;
   color: #000;
 }
+
+.note-btn-ctn {
```

Comment:
I think we might want to create a btn styles file. So all the button styles are in one place.